services/basket/src/main/java/org/cryptimeleon/incentive/services/basket/model/BasketItemNew.java

```diff
@@ -0,0 +1,35 @@
+package org.cryptimeleon.incentive.services.basket.model;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.cryptimeleon.incentive.services.basket.storage.ItemEntity;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class BasketItemNew {
```

Comment:
why the weird name?

Reply:
Good catch, this name was meant to exists only temporarily